package-lock.json

```diff
@@ -3289,6 +3290,10 @@
         "node": ">= 6"
       }
     },
+    "node_modules/placement_tracker_v2": {
+      "resolved": "",
+      "link": true
+    },
```

Comment:
Can you comment on the necessity of this dependency and remove it if not deemed necessary?

Reply:
This was unnecessary i have removed it